### PR TITLE
refactor: use tailwind native aspect-ratio utility class

### PR DESCRIPTION
**Pull request by emmanuelchucks**
I'm curious if there is any reason you chose to use the plugin instead of the utility class that comes with Tailwind.

Am I missing something?

### CODE REVIEW

**Review by leerob — APPROVED** [review]
Good call!

**Review by leerob — CHANGES_REQUESTED** [review]
Huh, interesting - actually seems like it's not working on the preview deploy.

**Comment by emmanuelchucks** [comment]
My bad. Should have used "position: relative" on Image's parent element

File: pages/index.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ function BlurImage({ image }: { image: Image }) {
 
   return (
     <a href={image.href} className="group">
-      <div className="aspect-w-1 aspect-h-1 w-full overflow-hidden rounded-lg bg-gray-200 xl:aspect-w-7 xl:aspect-h-8">
+      <div className="relative aspect-square w-full overflow-hidden rounded-lg bg-gray-200 xl:aspect-[7/8]">
```
Comment (leerob) [inline pages/index.tsx:48]:
```suggestion
      <div className="aspect-square w-full overflow-hidden rounded-lg bg-gray-200 xl:aspect-[7/8]">
```

File: pages/index.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ function BlurImage({ image }: { image: Image }) {
 
   return (
     <a href={image.href} className="group">
```
Comment (leerob) [inline pages/index.tsx:47]:
```suggestion
    <a href={image.href} className="group relative">
```

**Comment by emmanuelchucks** [comment]
"Position: relative" should be on direct parent of image to prevent covering up the name and twitter handle. I think there was some miscommunication, I had already made the necessary changes before my previous comment:

> My bad. Should have used "position: relative" on Image's parent element

Check my latest commit [da5c32a](https://github.com/leerob/image-gallery-supabase-tailwind-nextjs/pull/2/commits/da5c32a2d735314f87b14fadb520117df1c6910b)

**Pull request closed without merging**